.changeset/dry-forks-relate.md

```diff
@@ -0,0 +1,7 @@
+---
+'braid-design-system': minor
+---
+
+Add zero opacity to Box
+
+Provide zero opacity on `Box` as an optimisation.
```

Comment:
Probably worth adding an example usage here. Would also help make it clear to consumers that it's a number.